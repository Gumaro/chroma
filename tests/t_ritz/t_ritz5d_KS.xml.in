<?xml version="1.0"?>
<params>
  <!-- Lattice Dimensions etc -->
  <lattice>
    <nrow>4 4 4 8</nrow>
    <boundary>1 1 1 -1</boundary>
  </lattice>

  <!-- RNG Seed for initing Gaussians and the like -->
  <RNG>
    <seed>11 0 0 0</seed>
  </RNG>

  <!-- Start up config details -->
  <Cfg>
     <!-- This will probably soon be standardised. 2 means from file  -->
     <startType>2</startType>
     <gaugeFilename>../test_purgaug.cfg1</gaugeFilename>

     <!-- Format = 0 means SZIN format -->
     <gaugeFileFormat>0</gaugeFileFormat>
  </Cfg>
  <quarkMass>0.1</quarkMass>
  <eig>
     <RsdR>1.0e-4</RsdR>
     <RsdA>1.0e-6</RsdA>
     <RsdZero>5.0e-6</RsdZero>
     <ProjApsiP>false</ProjApsiP>    
     <Neig>2</Neig>
     <Ndummy>2</Ndummy>
     <gammaFactor>0.1</gammaFactor>
     <MaxKS>100</MaxKS>
     <MaxCG>10000</MaxCG>
  </eig>

  <zolotarev>
	<approxOrder>15</approxOrder>
	<approxMin>0.668</approxMin>
	<approxMax>6.9</approxMax>
	<wilsonMass>-1.4</wilsonMass>
  </zolotarev>
</params>
