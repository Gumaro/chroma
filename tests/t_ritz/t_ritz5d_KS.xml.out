<?xml version="1.0"?>


<ritzTest>
  <params>
    <lattice>
      <nrow>4 4 4 8</nrow>
      <boundary>1 1 1 -1</boundary>
    </lattice>
    <RNG>
      <seed>11 0 0 0</seed>
    </RNG>
    <quarkMass>0.1</quarkMass>
    <Cfg>
      <startType>2</startType>
      <gaugeFileFormat>0</gaugeFileFormat>
      <gaugeFilename>../test_purgaug.cfg1</gaugeFilename>
    </Cfg>
    <eig>
      <RsdR>1e-04</RsdR>
      <MaxCG>10000</MaxCG>
      <Neig>2</Neig>
      <Ndummy>2</Ndummy>
      <RsdA>1e-06</RsdA>
      <RsdZero>5e-06</RsdZero>
      <ProjApsiP>false</ProjApsiP>
      <gammaParam>0.1</gammaParam>
    </eig>
    <zolotarev>
      <approxOrder>15</approxOrder>
      <approxMin>0.668</approxMin>
      <approxMax>6.9</approxMax>
      <wilsonMass>-1.4</wilsonMass>
    </zolotarev>
  </params>
  <GaugeInfo><szin>
      <cfg_version>7</cfg_version>
      <date>Sat Jun 16 00:35:57 2001</date>
      <banner>(Sat Jun 16 00:21:07  2001) Source=MOTIE Target=linux Release= Quarks=g</banner>
      <FermTypeP>0</FermTypeP>
      <Nd>4</Nd>
      <Nc>3</Nc>
      <BetaMC>20.1</BetaMC>
      <BetaMD>0</BetaMD>
      <KappaMC>0</KappaMC>
      <KappaMD>0</KappaMD>
      <MassMC>0</MassMC>
      <MassMD>0</MassMD>
      <dt>0</dt>
      <MesTrj>1</MesTrj>
      <TotalCG>0</TotalCG>
      <TotalTrj>2</TotalTrj>
      <spec_acc>0</spec_acc>
      <NOver>2</NOver>
      <TotalTry>12288</TotalTry>
      <TotalFail>56</TotalFail>
      <Nf>9.48851e+33</Nf>
      <Npf>2</Npf>
      <RefMomTrj>302059064</RefMomTrj>
      <RefFnoiseTrj>36</RefFnoiseTrj>
      <LamPl>9.45468e+33</LamPl>
      <LamMi>0</LamMi>
      <AlpLog>4.2039e-45</AlpLog>
      <AlpExp>4.2039e-45</AlpExp>
      <nrow>4 4 4 8</nrow>
      <seed>
        <Seed>
          <elem>811</elem>
          <elem>1073</elem>
          <elem>1535</elem>
          <elem>1310</elem>
        </Seed>
      </seed>
    </szin>

  </GaugeInfo>
  <plaquette>
    <w_plaq>0.902263</w_plaq>
    <s_plaq>0.862579</s_plaq>
    <t_plaq>0.941947</t_plaq>
    <link>0.734641</link>
  </plaquette>
<Zolo5DInit>
  <ZoloContFracCoeff>
    <beta>34.8276 19.4581 13.2979 9.74636 7.38531 5.69973 4.44527 3.48441 2.73031 2.12388 1.62299 1.19677 0.822007 0.480697 0.158208</beta>
  </ZoloContFracCoeff>
  <ZoloEquivTransCoeff>
    <beta>1 1 1 1 1 1 1 1 1 1 1 1 1 1 0.158208</beta>
    <alpha>0.0384138 0.0621668 0.0878391 0.117868 0.15413 0.198666 0.254089 0.324212 0.415269 0.538614 0.717524 1.00822 1.59084 1.44233 0.158208</alpha>
  </ZoloEquivTransCoeff>
</Zolo5DInit>


<EigSpecRitzKS>
  <KS_iter>
    <ev>
      <n_count>5</n_count>
      <final_grad>0.166205</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>0.16669</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>0.167115</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>0.165795</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.0446987</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.0432151</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.0488377</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.0494287</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>19</n_count>
      <final_grad>0.0131359</final_grad>
    </ev>
    <ev>
      <n_count>19</n_count>
      <final_grad>0.0134057</final_grad>
    </ev>
    <ev>
      <n_count>18</n_count>
      <final_grad>0.0147008</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>0.0134176</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>163</n_count>
      <final_grad>0.00444452</final_grad>
    </ev>
    <ev>
      <n_count>172</n_count>
      <final_grad>0.00461428</final_grad>
    </ev>
    <ev>
      <n_count>194</n_count>
      <final_grad>0.00467866</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00589414</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>24</n_count>
      <final_grad>0.00124444</final_grad>
    </ev>
    <ev>
      <n_count>24</n_count>
      <final_grad>0.00126818</final_grad>
    </ev>
    <ev>
      <n_count>20</n_count>
      <final_grad>0.00145766</final_grad>
    </ev>
    <ev>
      <n_count>28</n_count>
      <final_grad>0.00171206</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00101654</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.0010314</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00200187</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00152139</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>94</n_count>
      <final_grad>0.000348271</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000582911</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00093011</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00252284</final_grad>
    </ev>
    <n_jacob>3</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000224236</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000420448</final_grad>
    </ev>
    <ev>
      <n_count>11</n_count>
      <final_grad>0.000278007</final_grad>
    </ev>
    <ev>
      <n_count>6</n_count>
      <final_grad>0.000625781</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>8</n_count>
      <final_grad>7.10365e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000449244</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000277973</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.00155087</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>8.31018e-05</final_grad>
    </ev>
    <ev>
      <n_count>40</n_count>
      <final_grad>0.000160564</final_grad>
    </ev>
    <ev>
      <n_count>80</n_count>
      <final_grad>8.79991e-05</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>0.000438932</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>10</n_count>
      <final_grad>2.26938e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000205814</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>8.6195e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000647708</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>3.34327e-05</final_grad>
    </ev>
    <ev>
      <n_count>80</n_count>
      <final_grad>6.09989e-05</final_grad>
    </ev>
    <ev>
      <n_count>6</n_count>
      <final_grad>2.39115e-05</final_grad>
    </ev>
    <ev>
      <n_count>9</n_count>
      <final_grad>0.000174051</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>1.51762e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>5.85126e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>4.18009e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>0.000325683</final_grad>
    </ev>
    <n_jacob>2</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>20</n_count>
      <final_grad>4.85737e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.80946e-05</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.49263e-05</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>8.63848e-05</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>5.81675e-06</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>1.92707e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>2.89017e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>9.23643e-05</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>14</n_count>
      <final_grad>1.8261e-06</final_grad>
    </ev>
    <ev>
      <n_count>10</n_count>
      <final_grad>5.22441e-06</final_grad>
    </ev>
    <ev>
      <n_count>90</n_count>
      <final_grad>2.78473e-05</final_grad>
    </ev>
    <ev>
      <n_count>6</n_count>
      <final_grad>9.45001e-06</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>200</n_count>
      <final_grad>2.38714e-06</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>6.86981e-06</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>3.80358e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>7.88604e-06</final_grad>
    </ev>
    <n_jacob>1</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>5</n_count>
      <final_grad>6.98038e-07</final_grad>
    </ev>
    <ev>
      <n_count>89</n_count>
      <final_grad>2.04086e-06</final_grad>
    </ev>
    <ev>
      <n_count>5</n_count>
      <final_grad>1.16311e-05</final_grad>
    </ev>
    <ev>
      <n_count>8</n_count>
      <final_grad>2.18039e-06</final_grad>
    </ev>
    <n_jacob>0</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>5</n_count>
      <final_grad>8.47628e-07</final_grad>
    </ev>
    <ev>
      <n_count>10</n_count>
      <final_grad>1.03783e-06</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>1.40498e-05</final_grad>
    </ev>
    <ev>
      <n_count>200</n_count>
      <final_grad>3.37734e-06</final_grad>
    </ev>
    <n_jacob>0</n_jacob>
  </KS_iter>
  <KS_iter>
    <ev>
      <n_count>5</n_count>
      <final_grad>5.49681e-07</final_grad>
    </ev>
    <ev>
      <n_count>6</n_count>
      <final_grad>8.79572e-07</final_grad>
    </ev>
    <ev>
      <n_count>6</n_count>
      <final_grad>4.25528e-06</final_grad>
    </ev>
    <ev>
      <n_count>19</n_count>
      <final_grad>1.06461e-06</final_grad>
    </ev>
    <n_jacob>0</n_jacob>
  </KS_iter>
  <final_n_jacob>0</final_n_jacob>
  <n_cg_tot>7895</n_cg_tot>
  <n_KS>20</n_KS>
  <lambda_H>0.00268499 0.00268652 1 1</lambda_H>
</EigSpecRitzKS>

  <lambda>0.00268499 0.00268652 1 1</lambda>
  <check_norm>5.49681e-07 8.79572e-07</check_norm>
  <check_norm_rel>0.000204723 0.000327401</check_norm_rel>
  <eigFix>
    <lambda>-0.051816 0.0518307 1 1</lambda>
    <n_valid>2</n_valid>
    <valid_eig>1 1</valid_eig>
    <check_norm>4.70901e-05 6.28322e-05</check_norm>
    <check_norm_rel>0.000908794 0.00121226</check_norm_rel>
  </eigFix>
  <szinLamda>-0.012638 0.0126416 1 1</szinLamda>
</ritzTest>